### PR TITLE
compatibility >= gitbucket-4.19, simplified build

### PR DESCRIPTION
**Pull request by McFoggy**
As I needed to give your plugin a trial on my own 4.22 installation, I updated it and adapted it to match standard builds from official plugins (like gitbucket-notifications-plugin).

### CODE REVIEW

**Comment by lefou** [comment]
Great, thank you! I will review you PR in the next days.

**Comment by lefou** [comment]
Thank you

File: project/plugins.sbt [hunk 1]
```diff
@@ -1 +1 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.0")
\ No newline at end of file
+addSbtPlugin("io.github.gitbucket" % "sbt-gitbucket-plugin" % "1.2.0")
```
Comment (kounoike) [inline project/plugins.sbt:1]: Now, you can use sbt-gitbucket-plugin 1.3.0. It adds GitBucket version to assembly jar filename.

Comment (lefou) [reply project/plugins.sbt:1]: I already bumped the version after the merge, but thanks for the hint.

**Pull request merged**